docs/grid-functionalities/Composite-Editor-Modal.md

```diff
@@ -265,6 +265,16 @@ this.compositeEditorInstance?.openDetails({
 });
 ```
 
+### onDispose
+The `onDispose` callback function notifies you that the editor is finished and being disposed. It gets called regardless of whether there are changes or not.
+```ts
+this.compositeEditorInstance?.openDetails({
+  headerTitle: 'Create Item',
+  modalType: 'create',
+  onDispose: () => console.log('editor removed'),
+});
+```
```

Comment:
that's good, though we also need to update all framework docs, so you should modify 5 docs in total (4 frameworks + 1 vanilla). You can easily find all other docs by doing a search across the entire project, for example search for this: "The `onError` callback function will execute anytime an error is thrown by the modal window."

Reply:
See 2988d95bd3f97cea488329ea383bf5b839dcf378